website/docs/components/app-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,93 @@
+## Usage
+
+### When to use
+
+- Provide consistent global navigation across an application.
+- Offer quick access to important utilities such as user settings, help, and search.
+- Establish a clear visual hierarchy and brand presence at the top of the application.
+
+### When not to use
+
+- To navigate within subpages of an organization or project; instead, use the [Side Nav](/components/side-nav).
+
+### Navigation hierarchy
+
+The App Header provides consistent navigation between global application areas, such as organizations and projects, while facilitating easy access to utilities like support and user settings.
+
+In the hierarchy of navigational elements, the App Header sits at the top and is followed by:
+
+- The [Side Nav](/components/side-nav), which focuses on contextual pages and subpages within a project or organization.
+- The [App Footer](/components/app-footer), which focuses on non-essential metadata and general links.
+
+![Example of the hierarchy of navigation elements](/assets/components/app-header/app-header-hierarchy.png)
+
+## Global navigation
+
+### Home link
+
+The home Link gives users quick, consistent access to the application's home page or dashboard.
+
+For cloud products, the icon should always be `hcp`. For standalone or open-source products, it should be the product’s service icon (e.g. `terraform`).
+
+![Examples of the Home link using different logos](/assets/components/app-header/app-header-home-link.png)
+
+### Context switcher
+
+The context switcher allows users to switch between different global contexts within the product or application, such as navigating between different organizations or projects. This element only exists in Figma. For implementation in code, use the HDS [Dropdown](/components/dropdown) or, if necessary, create your own custom component styled after the Dropdown.
+
+![Context switcher within the App Header](/assets/components/app-header/app-header-context-switcher.png)
+
+## Utility navigation
+
+### Help dropdown
+
+Use the help dropdown to provide users with access to support and helpful resources that can be easily accessible from anywhere within the application, e.g., links to the help center, documentation, or tutorials.
+
+Avoid placing non-help-related links within the help dropdown menu.
+
+![Help dropdown in the App Header](/assets/components/app-header/app-header-help-dropdown.png)
+
+### User dropdown
+
+The user dropdown gives users quick and easy access to their settings and preferences. The menu should contain links or actions related to the user's profile, settings, and/or preferences.
+
+Avoid placing links to unrelated pages within the user dropdown menu.
+
+![User dropdown in the App Header](/assets/components/app-header/app-header-user-dropdown.png)
```

Comment:
The text "Signed in with Github" and the username are using the incorrect list item types. The HDS recommended version looks like this: 

<img width="215" alt="Screenshot 2024-07-15 at 5 35 56 PM" src="https://github.com/user-attachments/assets/e887164d-8530-4673-b7f1-8c7a9bc90370">

Reply:
fixed this